Update knife_bootstrap.rst

jeremiahsnapp opened the pull request:
Files in `.chef/client.d` get copied to the bootstrapped system's `/etc/chef/client.d` directory

jamescott commented:
@delivery review

chef-delivery commented:
Failed to review change: bb773805-15fe-4dcd-8715-c42ddc1e985f @jamescott

coderanger commented:
I'm going to guess that's not something I can fix?

jamescott commented:
Hmm, yeah. It's the Delivery integration. 

https://github.com/chef/chef-web-docs/pull/698

jamescott commented:
It's merged, so should get picked up. I'll keep an eye on this one for you. Thanks for submitting!

james

jamescott commented:
Is fixed:

https://docs.chef.io/custom_resources.html#create-templates

Thanks Noah.

jeremiahsnapp committed:
Update knife_bootstrap.rst

Files in `.chef/client.d` get copied to the bootstrapped system's `/etc/chef/client.d` directory

jeremiahsnapp committed:
Update knife_bootstrap.rst

Commit:
fixes internal link

kagarmoe commented:
![tenor-174732225](https://cloud.githubusercontent.com/assets/4400151/26702455/ecdff9b2-46d9-11e7-8787-f450c9363da3.gif)
